src/templates/fr/2019/accessibility_statement.html

```diff
@@ -0,0 +1,123 @@
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="specifications-techniques"><a href="#specifications-techniques" class="anchor-link">Spécifications techniques</a></h2>
+        
+        <p>
+          Le site Web est construit sur du HTML sémantique bien structuré (y compris <a href="https://www.w3.org/WAI/standards-guidelines/aria/" hreflang="en">WAI-ARIA</a> le cas échéant), amélioré avec CSS pour le style et JavaScript limité pour certains composants interactifs. Le site Web utilise des pratiques de développement modernes telles que les polices CSS Grid, Fetch et WOFF2, qui ne sont pas prises en charge dans les navigateurs plus anciens tels que IE 11. Le site Web est toujours entièrement fonctionnel dans ces navigateurs mais peut ne pas être aussi attrayant visuellement. Aucun plug-in ou bibliothèque supplémentaire ne devrait être nécessaire pour afficher le site Web.
+        </p>
+        <p>
+          Le Web Almanac PDF suit les mêmes normes d’accessibilité que le site Web et, en outre, est structurellement étiqueté pour faciliter l’accessibilité.
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="approche-evaluation"><a href="#approche-evaluation" class="anchor-link">Approche d’évaluation</a></h2>
+        
+        <p>
+          Auto-évaluation&nbsp;: le contenu a été évalué par l’équipe Web Almanac en utilisant sa propre expertise et un certain nombre d’outils dont <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, le <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>. Le Web Almanac PDF a été testé avec une analyse d’accessibilité dans Acrobat Reader Pro DC.
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="retour-information"><a href="#retour-information" class="anchor-link">Retour d’information</a></h2>
+
+        <p>
+          Nous apprécions les commentaires sur l’accessibilité de ce site Web. La meilleure façon de nous joindre serait de <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">ouvrir un ticket sur Github</a> ou de nous envoyer un courriel à <a href="mailto:team@httparchive.org">team@httparchive.org</a>. Nous serions également ravis de recevoir des pull-requests pour ceux qui souhaitent contribuer directement. Le Web Almanac est un projet communautaire, géré par des bénévoles, et nous ne pouvons donc pas garantir un délai pour répondre aux commentaires, mais nous y répondrons dès que possible.
```

Comment:
Inclusivité en changeant de voix (évite de faire de "nous" un masculin).

```suggestion
          Vos commentaires sur l’accessibilité de ce site Web nous intéressent. La meilleure façon de nous joindre serait <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues" hreflang="en">d’ouvrir un ticket sur Github</a> ou de nous envoyer un courriel à <a href="mailto:team@httparchive.org">team@httparchive.org</a>. Vous souhaitez contribuer directement ? N'hésitez pas à proposer vos <span lang="en">Pull Requests<span>, elles seront les bienvenues. Le Web Almanac est un projet communautaire, géré par des bénévoles, et nous ne pouvons donc pas garantir un délai pour répondre aux commentaires, mais nous y répondrons dès que possible.
```

Reply:
Excellente cette reformulation (et les autres aussi), merci !